### ISSUE TITLE
dont run pwitness

### ISSUE THREAD

**Issue by kaankozan** [tronprotocol/java-tron#793]
Please answer these questions before submitting your issue. Thanks!

install java 8 oracle
```
root@tron-1:~/java-tron# java -version
java version "1.8.0_171"
Java(TM) SE Runtime Environment (build 1.8.0_171-b11)
Java HotSpot(TM) 64-Bit Server VM (build 25.171-b11, mixed m
ode)
root@tron-1:~/java-tron#
```
```
root@tron-1:~/java-tron# echo $JAVA_HOME
/usr/lib/jvm/java-8-oracle
```
```
root@tron-1:~# javac -version
javac 1.8.0_171
root@tron-1:~#
```
i run `./gradlew build -x test` (test aborting because build failed)
Build success

Now copy to config.file in java-tron
```
root@tron-1:~/java-tron# cp src/main/resources/config.conf c
onfig.conf
```
And i edit config.conf in localwitness my private key

And i run `./gradlew run -Pwitness`

But i have a failed message 

```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':run'.
> Process 'command '/usr/lib/jvm/java-8-oracle/bin/java'' fi
nished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run wit
h --info or --debug option to get more log output.

* Get more help at https://help.gradle.org

BUILD FAILED in 6s
6 actionable tasks: 1 executed, 5 up-to-date
```

What is wrong 
Can you help me pls i try 5 days but not running...

**Comment by bondibox** [comment]
> i run ./gradlew build -x test (test aborting because build failed)

Let me guess: You're using Ubuntu.

**Comment by bondibox** [comment]
I thought so. I opened issue #788 which links to an issue I reported at the gradle project. I think the linux builds are breaking because the gradle libraries (300 of them) are constantly getting updated and it's hit or miss as to whether that will break the java-tron build. But the gradle people didn't think it was a clearly reported bug with reproduceable results (install linux -> clone java-tron -> ./gradlew build -> not clear enough for gradle team) so they closed the issue. Hopefully the java-tron team will care more about it.
